google-cloud-bigquery/lib/google/cloud/bigquery/dataset.rb

```diff
@@ -1435,16 +1457,52 @@ def routines token: nil, max: nil, filter: nil
         #
         # @!group Data
         #
-        def query_job query, params: nil, types: nil, external: nil, priority: "INTERACTIVE", cache: true, table: nil,
-                      create: nil, write: nil, dryrun: nil, standard_sql: nil, legacy_sql: nil, large_results: nil,
-                      flatten: nil, maximum_billing_tier: nil, maximum_bytes_billed: nil, job_id: nil, prefix: nil,
-                      labels: nil, udfs: nil
+        def query_job query,
+                      params: nil,
+                      types: nil,
+                      external: nil,
+                      priority: "INTERACTIVE",
+                      cache: true,
+                      table: nil,
+                      create: nil,
+                      write: nil,
+                      dryrun: nil,
+                      standard_sql: nil,
+                      legacy_sql: nil,
+                      large_results: nil,
+                      flatten: nil,
+                      maximum_billing_tier: nil,
+                      maximum_bytes_billed: nil,
+                      job_id: nil,
+                      prefix: nil,
+                      labels: nil,
+                      udfs: nil,
+                      create_session: nil,
```

Comment:
Should this default to `false` rather than `nil`?

Reply:
Great question! We *try* to never provide defaults (other than `nil`) for parameters that are pass-through in the client (not used in the client.) The reason for this is the same as not providing client-side validation for pass-through arguments. By relying on server-side default values and validations, we provide the same functionality to users (at the cost of an RPC call in the case of validation errors) while avoiding potential conflicts or breaking code changes in the client if the service behavior changes. I wrote "try" above because there are a few regrettable aberrations, such as `priority: "INTERACTIVE"` in this method.

However, I now see that I forgot to document the service default value for this param. I'll add it now. Thanks for the heads-up.